doc-content/enterprise-only/project-data/preview-editor-con.adoc

```diff
@@ -0,0 +1,11 @@
+[id='preview-editor-con']
+= Test Scenario (Preview) editor
+
+{PRODUCT} now comes with a new *Test Scenario (Preview)* editor. The preview editor provides a new and enhanced user experience when you test or validate your business rules and data.
```

Comment:
"  {PRODUCT} now comes with a new *Test Scenario (Preview)* editor. "
Has a marketing tone. Suggestion:  Skip the first sentence, and say:
The new *Test Scenario (Preview)* editor provides a new and enhanced user experience when you test or validate your business rules and data.

Also, are we mentioning somewhere that for this release both the editors are available ?
'

Reply:
No change. Test Scenario (Preview) is a UI element.